src/auth/auth.py

```diff
@@ -80,7 +80,7 @@ async def check_authentication(
 
     if session_id:
         await crud.task_clean_expired_user_sessions(db_session)
-        response_dict = await crud.check_session_validity(db_session, session_id)
+        response_dict = await crud.check_user_validity(db_session, session_id)
```

Comment:
Should this function be `crud.check_user_session`?

Reply:
yes! good catch